packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -473,7 +480,7 @@ describe('Collection aggregations tab', function () {
     const confirmDuplicateButton = await browser.$(
       Selectors.DuplicateViewModalConfirmButton
     );
-    confirmDuplicateButton.waitForEnabled();
+    await confirmDuplicateButton.waitForEnabled();
```

Comment:
Something about webdriverio's old types caused typescript to not notice that this is a promise that wasn't being awaited. Now it works as expected with the latest webdriver.